configure

```diff
@@ -11,7 +11,5 @@ try() {
     fi
 }
 
-try python2.7 "$@"
-try python27 "$@"
-try python2 "$@"
-exec python $script "$@"
+try python3 "$@"
+exec python3 $script "$@"
```

Comment:
It might make sense to fallback on python 2 here to avoid breaking compatibility, i.e., we just try python 3 *first*.